pkgs/by-name/so/solhint/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+}:
+
+buildNpmPackage {
```

```suggestion
buildNpmPackage (finalAttrs: {
```
This'll help you reuse some attrs